### PR TITLE
Python redo

### PR DESCRIPTION
**Pull request by dturanski**
This uses smaller base image `python2.7:slim` for python2-function-invoker.

### CODE REVIEW

File: samples/python/sentiments/Dockerfile [hunk 1]
```diff
@@ -1,2 +1,7 @@
-FROM sk8s/python2-function-invoker:0.0.1-SNAPSHOT
-ADD sentiment_service.py /functions/function.py
+FROM sk8s/sk8s-python2-function-invoker:0.0.1-SNAPSHOT
```
Comment (markfisher) [inline samples/python/sentiments/Dockerfile]: need to remove the prefix now

File: faas [hunk 2]
```diff
@@ -134,6 +134,7 @@ function apply() {
 function delete() {
   echo "Deleting $FUNCTION resource[s]"
   kubectl delete function $FUNCTION
+  kubectl delete all -l function=$FUNCTION
```
Comment (markfisher) [inline faas]: this should be handled already by: https://github.com/markfisher/sk8s/blob/master/event-dispatcher/src/main/java/io/sk8s/event/dispatcher/EventDispatchingHandler.java#L66

Comment (dturanski) [reply faas]: That's what @trisberg says. It wasn't working for me.  I'll take it out.

**Comment by dturanski** [comment]
Closing this due to merging difficulties.  Will create a new one.

**Pull request closed without merging**